cypress/e2e/swap/errors.test.ts

```diff
@@ -116,4 +116,21 @@ describe('Swap errors', () => {
       getBalance(DAI).should('be.closeTo', initialBalance + 200, 1)
     })
   })
+
+  it('insufficient liquidity', () => {
+    cy.visit(`/swap?inputCurrency=${USDC_MAINNET.address}&outputCurrency=${DAI.address}`)
+    // The API response is too variable so stubbing a 404.
+    cy.intercept('POST', 'https://api.uniswap.org/v2/quote', (req) => {
+      req.reply({
+        statusCode: 404,
+        fixture: 'insufficientLiquidity.json',
+        headers: {
+          'access-control-allow-origin': '*',
+        },
+      })
```

Comment:
I think this can be simplified to not use an anonymous function, see https://docs.cypress.io/api/commands/intercept#With-a-fixture.

Reply:
also, try moving the stub to before the `visit`. That way if it runs a quote on load it will still be intercepted.